ext/openssl/ossl_ssl.c

```diff
@@ -1046,6 +1046,26 @@ ossl_sslctx_set_ciphers(VALUE self, VALUE v)
     return v;
 }
 
+/*
+ * call-seq:
+ *    ctx.enable_fallback_scsv() => nil
+ *
+ * Activate TLS_FALLBACK_SCSV for this context.
+ * See RFC 7507.
+ */
+#ifdef SSL_MODE_SEND_FALLBACK_SCSV
```

Comment:
I think ossl_sslctx_enable_fallback_scsv() should be placed after ossl_sslctx_set_security_level() so the ordering match with the rb_define_method() lines. (We haven't been always consistent, but ...)

By the way, RDoc seems to fail to find the doc. Moving `#ifdef SSL_MODE_SEND_FALLBACK_SCSV` above the block comment fixes it for me. This might be considered a bug of RDoc's C parser, though.